Add work type

markpbaggett opened the pull request:
## What Does This Do

1. Adds new work_type property.
2. Updates definitions for data providers.

## How Do I Test

Do tests pass?

markpbaggett committed:
Add better definitions.

markpbaggett committed:
Add work type.

markpbaggett committed:
Add multivalue property.

markpbaggett committed:
Fix indentation.

mlhale7 commented on maps/utk.yml:
```diff
@@ -5955,6 +5956,38 @@ properties:
+      default: Describes the local work type of the work.
+    display_label:
+      default: Work Type
+    index_documentation: displayable, searchable
```
I don't think "index_documentation" actually does anything, but I don't think we want it to be able to be displayed.

mlhale7 commented on maps/utk.yml:
```diff
@@ -5955,6 +5956,38 @@ properties:
+    display_label:
+      default: Work Type
+    index_documentation: displayable, searchable
+    indexing:
```
Do we want "indexing" to be "admin_only"?

mlhale7 reviewed (commented):
Just a few questions on indexing . . .

markpbaggett committed:
Switch indexing.

markpbaggett committed:
Switch title.

markpbaggett committed:
Switch to has_work_type.

Upon further inspection, it appears work_type is a reserved
property.  This causes import to break with a NoMethodError -
undefined method `method_defined?' for nil:NilClass.
